custom/install/install_hooks.php

```diff
@@ -0,0 +1,80 @@
+<?php
```

Comment:
We can not add anything to the custom folder as it should be upgrade-safe

Reply:
Hi, 
changes already done.
I remove this file from the custom directory.

Regards